packages/mobile/src/app/reducers.ts

```diff
@@ -85,6 +86,7 @@ const initialState = {
   multiTokenUseUpdatedFeed: REMOTE_CONFIG_VALUES_DEFAULTS.multiTokenUseUpdatedFeed,
   linkBankAccountEnabled: REMOTE_CONFIG_VALUES_DEFAULTS.linkBankAccountEnabled,
   sentryTracesSampleRate: REMOTE_CONFIG_VALUES_DEFAULTS.sentryTracesSampleRate,
+  dappsExplorerEnabled: REMOTE_CONFIG_VALUES_DEFAULTS.dappsExplorerEnabled,
```

Comment:
nit: Can we put this variable next to `dappListApiUri `? Since both are related to the same thing :)

Reply:
I think we also need to add `dappsExplorerEnabled` to the reducer for `Actions.UPDATE_REMOTE_CONFIG_VALUES` as well